agent/stats/engine.go

```diff
@@ -604,26 +606,34 @@ func (engine *DockerStatsEngine) taskContainerMetricsUnsafe(taskArn string) ([]*
 			continue
 		}
 
-		networkStatsSet, err := container.statsQueue.GetNetworkStatsSet()
+		containerMetric := &ecstcs.ContainerMetric{
+			ContainerName:  &container.containerMetadata.Name,
+			CpuStatsSet:    cpuStatsSet,
+			MemoryStatsSet: memoryStatsSet,
+		}
+
+		task, err := engine.resolver.ResolveTask(dockerID)
 		if err != nil {
-			// we log the error and still continue to publish cpu, memory stats
-			seelog.Warnf("Error getting network stats: %v, container: %v", err, dockerID)
+			seelog.Warnf("Task not found for container ID: %s", dockerID)
+		} else {
+			// send network stats for default/bridge/nat network modes
```

Comment:
Do we need to log when the customer is using `host`/`none` network modes, (if it's not in the logs already)?  We might need this to troubleshoot logs out of context.

Reply:
We already log as debug here - https://github.com/aws/amazon-ecs-agent/blob/network-disk-metrics/agent/stats/utils.go#L57 
Adding another log statement may make the logs too messy since these will be logged once every 20 seconds.